_partials/_devops-rest-api-get-started.md

```diff
@@ -0,0 +1,176 @@
+
+[$REST_LONG][rest-api-reference] is a comprehensive RESTful API you use to manage $CLOUD_LONG resources
+including VPCs, services, and read replicas.
+
+This page shows you how to set up secure authentication for the $REST_LONG and create your first service.
+
+## Prerequisites
+
+<RESTPrereqs />
+
+* Install [curl][curl].
+
+
+## Configure secure authentication
+
+$REST_LONG uses HTTP Basic Authentication with access keys and secret keys. All API requests must include
+proper authentication headers.
+
+<Procedure> 
+
+1. **Set up API credentials**
+
+    1. In $CONSOLE [copy your project ID][get-project-id] and store it securely using an environment variable:
+
+      ```bash
+      export TIGERDATA_PROJECT_ID="your-project-id"
+      ```
+
+    1. In $CONSOLE [create your client credentials][create-client-credentials] and store them securely using environment variables:
+
+       ```bash
+       export TIGERDATA_ACCESS_KEY="Public key"
+       export TIGERDATA_SECRET_KEY="Secret key"
+       ```
+
+1. **Configure the API endpoint**
+
+   Set the base URL in your environment:
+
+    ```bash
+    export API_BASE_URL="https://console.cloud.timescale.com/public/api/v1"
+    ```
+
+1. **Test your authenticated connection to $REST_LONG by listing services**
+
+    ```bash
+    curl -X GET "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
+      -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
+      -H "Content-Type: application/json"
+    ```
+
+   This call returns something like:
+    - No services:
+      ```terminaloutput
+      []%
+      ```
+    - One or more services:
+
+      ```terminaloutput
+      [{"service_id":"a59clooxoe","project_id":"c8nmagk8zh","name":"events",
+      "region_code":"eu-central-1","service_type":"TIMESCALEDB",
+      "created":"2025-09-09T08:37:15.816443Z","paused":false,"status":"READY",
+      "resources":[{"id":"101228","spec":{"cpu_millis":500,"memory_gbs":2,"volume_type":""}}],
+      "metadata":{"environment":"DEV"},"endpoint":{"host":"oh.yeah.tsdb.cloud.timescale.com",
+      "port":12345}}] 
+      ```
+
+</Procedure>
+
+
+## Create your first service
+
+Create a new database service using the $REST_LONG with secure configuration.
+
+<Procedure>
+
+1. **Define the service creation payload in a JSON file**
+   ```bash
+   cat > service-config.json << 'EOF'
+   {
+     "name": "my-first-service",
+     "service_type": "TIMESCALEDB",
+     "region_code": "us-east-1",
+     "replica_count": 1,
+     "cpu_millis": 1000,
+     "memory_gbs": 4
+   }
+   EOF
+   ```
+
+1. **Submit service creation request**
+
+    1. Create the service using the POST endpoint:
+       ```bash
+       curl -X POST "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
+         -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
+         -H "Content-Type: application/json" \
+         -d @service-config.json
```

Comment:
While saving the request payload to a file and referencing it here does work, it seems a little complex/roundabout to me. I think it would more common to just include the payload inline in the `curl` request:


```suggestion
       curl -X POST "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
         -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
         -H "Content-Type: application/json" \
         -d '{
             "name": "my-first-service",
             "service_type": "TIMESCALEDB",
             "region_code": "us-east-1",
             "replica_count": 1,
             "cpu_millis": 1000,
             "memory_gbs": 4
         }'
```